apps/src/blockly/components/ModalFunctionEditor.jsx

```diff
@@ -28,22 +29,27 @@ export default function ModalFunctionEditor() {
             type="button"
             id={MODAL_EDITOR_DELETE_ID}
             onClick={emptyOnClick}
-            color={Button.ButtonColor.brandSecondaryDefault}
+            color={Button.ButtonColor.white}
+            style={buttonStyles}
             size={buttonSize}
-          >
-            {msg.delete()}
-          </Button>
+            text={msg.delete()}
+          />
           <Button
             type="button"
             id={MODAL_EDITOR_CLOSE_ID}
             onClick={emptyOnClick}
-            color={Button.ButtonColor.brandSecondaryDefault}
+            color={Button.ButtonColor.white}
+            style={buttonStyles}
             size={buttonSize}
-          >
-            {msg.closeDialog()}
-          </Button>
+            text={msg.closeDialog()}
+          />
         </div>
       </div>
     </div>
   );
 }
+
+const buttonStyles = {
```

Comment:
Open to suggestions about whether there is a better place for these style rules to live.

Reply:
Is there a reason they could not go int the css module?

Reply:
I can add them to the module, but couldn't figure out how to get the style object imported. Something like `moduleStyles.button` just lets me access the class name and doesn't seem to let me apply inline styles directly.

Reply:
Button takes [className](https://github.com/code-dot-org/code-dot-org/blob/bee0868b0e1dc11d26c6224c650bbe7d0c7c03f8/apps/src/templates/Button.jsx#L48) as a parameter, can we do that?

Reply:
Providing the class name _works_ in that the class ends up on the element, but each of its style rules gets clobbered by some other classes that are getting added by the Button component. It feels risky to try to change that component for this. Another option is that I could use `!important` to make sure the rules get used, but I've heard that's not really a best practice.

Reply:
oh gotcha, inline seems fine, maybe just add a comment explaining why we need an inline style.